TYPO3.Media/Documentation/References/Validators.rst

```diff
@@ -17,12 +17,12 @@ Supported validator options are (array)allowedOrientations with one or two out o
 
 *Example*::
 
-  [at]Flow\Validate("$image", type="\TYPO3\Media\Validator\ImageOrientationValidator",
+  [at]Flow\Validate("$image", type="\Neos\Media\Validator\ImageOrientationValidator",
```

Comment:
The references are generated automatically, they should not be edit manually anyways…